challenge/twitter.html

```diff
@@ -0,0 +1,382 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Not Twitter</title>
+    <link rel="stylesheet" href="twitter-left.css">
+    <link rel="stylesheet" href="twitter-middle.css">
+    <link rel="stylesheet" href="twitter-right.css">
+    <link rel="stylesheet" href="twitter-global.css">
+</head>
+
+<body>
+    <div class="container">
+
+        <!-- Left menu bar -->
+        <nav class="left">
```

Comment:
A nav tag into an unordered list, into lists, with svgs and anchors - all for navigation... textbook! Nicely done!